anomaly_detector/model/sompy_model.py

```diff
@@ -0,0 +1,39 @@
+
+from .base_model import BaseModel
+
+import os
+import numpy as np
+import logging
+import sompy
+
+_LOGGER = logging.getLogger(__name__)
+
+class SOMPYModel(BaseModel):
+    "SOMPY alternative SOM implementation with parallelization"
+
+    def train(self, inp, map_size, iterations):
+        mapsize = [map_size, map_size]
+        som = sompy.SOMFactory.build(inp, mapsize, mask=None, mapshape='planer',
```

Comment:
Would it make sense to add comment to document what all these parameters mean and how they were chosen?

Reply:
These are all actually the default values, so I just removed them here.